eng/build.yml

```diff
@@ -135,6 +137,12 @@ jobs:
         displayName: Build
         env:
           BuildConfig: $(_BuildConfig)
+
+    - ${{ if eq(parameters.agentOs, 'Darwin') }}:
```

Comment:
Ubuntu AOT testing isn't currently possible due to an out of date python in the CI image which doesn't support f-string. Consequently the `emcc.py` script is failing during AOT. This bit should be reverted when CI Ubuntu image is upgraded to 20.04 LTS.